packages/flutter/test/material/bottom_navigation_bar_test.dart

```diff
@@ -2492,6 +2505,7 @@ void main() {
 
 Widget boilerplate({ Widget? bottomNavigationBar, required TextDirection textDirection }) {
   return MaterialApp(
+    theme: ThemeData(useMaterial3: false),
```

Comment:
If most of the tests that use this function fail when `useMaterial3` is true by default, then this change is fine. If not. it would be better to redefine it like this and only set the new parameter when it's necessary:
```dart
Widget boilerplate({ Widget? bottomNavigationBar, required TextDirection textDirection, bool? useMaterial3 }) {
  return MaterialApp(
    theme: ThemeData(useMaterial3: useMaterial3),
    home: Localizations(
    ...
```